maquetado/index.html

```diff
@@ -8,37 +8,41 @@
     <link rel="stylesheet" type="text/css" href="../application.css">
 </head>
 <body>
+  <div class="navbar">
+    <img class="nav-logo" src="./assets/wolox_logo.svg" alt="">
+    <p class="nav-text">Developer Tools</p>
+  </div>
   <div class="content-title">
     <img src="./assets/triangle.svg" alt="" class="logo-title">
     <h1 class="title">Front-End Tools</h1>
   </div>
   <div class="content-tech">
     <div class="tech">
-      <img src="./assets/react_icon.svg" alt="" class="tech-image">
+      <img src="../maquetado/assets/react_icon.svg" alt="" class="tech-image">
```

Comment:
You dont need to reference `maquetado` folder.
You can just do `src="assets/react_icon.svg"`. Also, you should give this image an alt attribute.
Apply this comment to every image